### ISSUE TITLE
Training Loss Doesn't Improve for CIFAR10

### ISSUE THREAD

**Issue by QiyaoWei** [HobbitLong/CMC#62]
Dear authors,

Hope this question finds you well. As I was extending CMC to CIFAR10, I noticed that the model doesn't seem to be training, and l_p and ab_p numbers don't seem to be changing either. This is strange, as I never changed the training code. Please see below for a screenshot of this problem. What might be a cause for this? I am aware of #50, but this problem persists even after tuning the softmax and sampling parameters.

<img width="566" alt="Screen Shot 2021-01-31 at 11 49 54 AM" src="https://user-images.githubusercontent.com/36057290/106386919-41240280-6412-11eb-9173-bb1bbf96d7a3.png">
